tuna/driver_conv.py

```diff
@@ -270,3 +270,12 @@ def decompose_weight_t(self, db_obj):
   def set_cmd(self, data_type):
     """Set cmd based on tensor data type"""
     self.cmd = PREC_TO_CMD[ConfigType.convolution][data_type]
+
+  def __eq__(self, other):
+    if not isinstance(other, DriverConvolution):
+      return NotImplemented
+
+    if self.vars() == other.vars():
+      return True
+
+    return False
```

Comment:
why does this need to overwrite the driver_base eq function? It seems to check the same things.

Reply:
I see, perhaps this isn't necessary.